src/dotnet/commands/dotnet-run/RunCommand.cs

```diff
@@ -119,14 +121,15 @@ private void Initialize()
 
                 if (projectFiles.Length == 0)
                 {
+                    var project = "--project";
+
                     throw new InvalidOperationException(
-                        $"{LocalizableStrings.RunCommandInvalidOperationException1} {directory}." + Environment.NewLine +
-                        LocalizableStrings.RunCommandInvalidOperationException2);
+                        $"Couldn\'t find a project to run. Ensure a project exists in  {directory}, or pass the path to the project using {project}");
```

Comment:
This and the message below are messages to the user - so they should be localized.

Given our work with exceptions, the exception type should probably change here too.

(nit) Apostrophes don't need to be escaped.